packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/components/OauthProviders.astro

```diff
@@ -96,7 +96,7 @@ const {
 )}
 
 {SHOW_PROVIDER_ERROR && (
-        <div class="bg-base-300 rounded-lg p-4">
-            <p class="text-center">No Login provider configured. Please contact your administrator.</p>
-        </div>
+    <div class="bg-base-300 rounded-lg p-4" style="text-align:center">
```

Comment:
`style` attribute?? Does UnoCSS not have a `text-center`??

Reply:
I don't know why, but with specifically the Shoelace buttons this is the easiest way to center them without using a `<center>` tag

Reply:
never mind... issue resolved itself... swapped in [27a6991](https://github.com/astrolicious/studiocms/pull/156/commits/27a6991667da335a1af5a8fabae7f28eb59d771e)